src/ConnectedMode.UnitTests/Binding/BindingProcessImplTests.cs

```diff
@@ -230,10 +230,12 @@ public async Task DownloadQualityProfile_WhenQualityProfileIsNotAvailable_OtherL
                 logger: logger);
 
             var notifications = new ConfigurableProgressStepExecutionEvents();
-            var progressAdapter = new FixedStepsProgressAdapter(notifications);
+            var progressAdapter = new Mock<IProgress<FixedStepsProgress>>();
+            progressAdapter.Setup(x => x.Report(It.IsAny<FixedStepsProgress>()))
+                .Callback<FixedStepsProgress>(x => ((IProgressStepExecutionEvents)notifications).ProgressChanged(x.Message, x.CurrentStep));
```

Comment:
Tweaked the mock to forward the progress reporting calls to the `notifications` helper which will record them.